astropy/stats/tests/test_funcs.py

```diff
@@ -9,12 +9,19 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from astropy.utils import minversion
+
 try:
     import scipy  # pylint: disable=W0611
 except ImportError:
     HAS_SCIPY = False
+    SCIPY_LT_1_4 = False
 else:
     HAS_SCIPY = True
+    if minversion(scipy, '0.14.0'):
```

Comment:
By analogy with the various `NUMPY_LT_*` variables, I would stick with `LT` even though that means you'd test with `not SCIPY_LT_0_14` below.